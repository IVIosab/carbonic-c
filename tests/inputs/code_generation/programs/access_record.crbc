routine main () : integer is 
    var z : integer is 8;
    var x : record 
        var a : integer is 3;
        var b : integer is 7;
    end;

    print(x.a);
    print(x.b);
    x.a := 9;
    print(x.a);
    print(z);
    x.b := z;
    z := z + 10;
    print(x.b + z);

    return 0;
end