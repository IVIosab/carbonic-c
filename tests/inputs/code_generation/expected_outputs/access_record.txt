3
7
9
8
26
